mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/annotations/Entity.java

```diff
@@ -74,4 +74,24 @@
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.CLASS)
-public @interface Entity {}
+public @interface Entity {
+  /**
+   * Specifies a default keyspace to use when doing operations on a given {@code @Entity}
```

Comment:
```suggestion
   * Specifies a default keyspace to use when doing operations on this entity.
```
Javadoc doesn't need to link to itself.